plugins/notification-resources/src/utils.ts

```diff
@@ -172,3 +175,25 @@ export async function markAsUnread (object: DocUpdates): Promise<void> {
     txes
   })
 }
+
+export async function getDirectChannel (
+  client: TxOperations,
+  me: Ref<PersonAccount>,
+  employeeAccount: Ref<PersonAccount>
+): Promise<Ref<DirectMessage>> {
+  const accIds = [me, employeeAccount].sort()
+  const existingDms = await client.findAll(chunter.class.DirectMessage, {})
+  for (const dm of existingDms) {
+    if (deepEqual(dm.members.sort(), accIds)) {
```

Comment:
Probably we need sort it one time when creation/migration and don't call it every time?

Reply:
Yeah, the sorting of IDs is done when the space is created below, removed sorting here, thanks